sdk/src/main/scala/com/horizen/mainchain/api/MainchainApiRequests.scala

```diff
@@ -45,7 +46,7 @@ object CertificateRequestCreator {
              endEpochCumCommTreeHash: Array[Byte],
              proofBytes: Array[Byte],
              quality: Long,
-             withdrawalRequestBoxes: Seq[WithdrawalRequestBox],
+             withdrawalRequestBoxes: Seq[BackwardTransfer],
```

Comment:
rename `withdrawalRequestBoxes` to backwardTransfers`

Reply:
Done